source/genericobject/index.rst

```diff
@@ -479,7 +478,9 @@ These fields will be usable only by Biomedical's objects :
    
 .. warning::
 
-      Trailing ``s_id`` is mandatory in ``[plugin_genericobject_field*s_id*]``. See https://github.com/pluginsGLPI/genericobject/issues/93
+      Trailing ``s_id`` is mandatory in ``[plugin_genericobject_field*s_id*]`` because the GLPI framework requires
+      names between brackets to end with ``s_id``. In database, ``glpi_plugin_genericobject_fields`` is table name and ``id``, its foreign key.
+      See explanation on `GitHub <https://github.com/pluginsGLPI/genericobject/issues/93>`_.
```

Comment:
As said on IRC channel; I guess it would be better to link to the develop docs where naming are explained.